content/projects/gsoc/2020/project-ideas/jenkins-rest-plugin.adoc

```diff
@@ -4,15 +4,65 @@ title: "Jenkins REST Plugin"
 goal: "Create a new plugin to give Jenkins users the ability to make calls to other Jenkins instances via their REST API"
 category: Plugin
 year: 2020
-status: discussion
+status: draft
 skills:
 - Java
 - REST API
 - Jenkins Pipeline
 mentors:
-- "TBD"
+- "Sagar Utekar"
```

Comment:
This line takes a user id. The user id must be created under [content/_data/authors](https://github.com/jenkins-infra/jenkins.io/tree/master/content/_data/authors), for example the file could be named `sagar-utekar.adoc`.

Then you would put your name like this:
```
mentors:
- "sagar-utekar"
```

Reply:
Thanks for the help @martinda .

I have raised another pr for mentor profile here : https://github.com/jenkins-infra/jenkins.io/pull/2966

Reply:
You can ping your PRs in the [gitter gsoc channel](https://gitter.im/jenkinsci/gsoc-sig#).